set.mm

```diff
@@ -215215,6 +215183,40 @@ is an important property of monoids (see ~ mndid ), and therefore also for
       USVCURNECVBUOBUMUTUMDCEUTERVAUNUMUTEUMDUCUDUEUFUHUIUJUK $.
   $}
 
+  ${
+    $d B x $.  $d L x $.  $d R x $.  $d .+ x $.
+    lidrideqd.l $e |- ( ph -> L e. B ) $.
+    lidrideqd.r $e |- ( ph -> R e. B ) $.
+    lidrideqd.li $e |- ( ph -> A. x e. B ( L .+ x ) = x ) $.
+    lidrideqd.ri $e |- ( ph -> A. x e. B ( x .+ R ) = x ) $.
+    $( If there is a left and right identity element for any binary operation
+       (group operation) ` .+ ` , both identity elements are equal.
+       Generalization of statement in [Lang] p. 3: it is sufficient that "e" is
+       a left identity element and "e``" is a right identity element instead of
+       both being (two-sided) identity elements.  (Contributed by AV,
+       26-Dec-2023.) $)
+    lidrideqd $p |- ( ph -> L = R ) $=
+      ( co cv wceq oveq1 id eqeq12d rspcdva oveq2 eqtr3d ) AFEDKZFEABLZEDKZUAMT
+      FMBCFUAFMZUBTUAFUAFEDNUCOPJGQAFUADKZUAMTEMBCEUAEMZUDTUAEUAEFDRUEOPIHQS $.
+
+    $d B x y $.  $d L y $.  $d R y $.  $d G y $.  $d .+ y $.  $d .0. y $.
+    $d ph y $.
+    lidrideqd.b $e |- B = ( Base ` G ) $.
+    lidrideqd.p $e |- .+ = ( +g ` G ) $.
+    lidrididd.o $e |- .0. = ( 0g ` G ) $.
+    $( If there is a left and right identity element for any binary operation
+       (group operation) ` .+ ` , the left identity element (and therefore also
+       the right identity element according to ~ lidrideqd ) is equal to the
+       two-sided identity element.  (Contributed by AV, 26-Dec-2023.) $)
```

Comment:
Proposal:

> In a unital magma, any left identity is equal to the identity.

Reply:
A unital magma is not required a priori.